lib/flex_ui/widgets/ecommerce/product_list_item.dart

```diff
@@ -49,9 +50,9 @@ class ProductListItem extends StatelessWidget {
                 child: Column(
                   crossAxisAlignment: CrossAxisAlignment.start,
                   children: [
-                    Text(
-                      product.name,
-                      style: Theme.of(context).textTheme.bodyLarge,
+                    EmphasisText(
+                      htmlString: product.name,
+                      style: EmphasisTextStyle.italic,
```

Comment:
The only change for this PR is to have this be `bold` as well (consistency to match Search Page)

Reply:
Sure? web doesn't do it. I tried and... iissh it's quite aggresive to the eye 😬

Reply:
Well, IDK actually I may not be that bad. My preference is for no emphasis at all actually, just classic text
<img width="366" alt="image" src="https://github.com/BASE1com/flex-storefront/assets/4425455/65136d03-d162-41ec-bba4-7391a39d3450">

Reply:
Merged with `invertedBold` 👍

Reply:
Yea, you're right. Classic no emphasis would be best 😄  Let's change in a future UI cleanup